tests/kernel/xip/testcase.yaml

```diff
@@ -1,5 +1,5 @@
 tests:
   arch.common.xip:
     arch_exclude: xtensa
-    platform_exclude: cc3220sf_launchxl
+    platform_exclude: cc3220sf_launchxl minnowboard
```

Comment:
FWIW: there's a filter syntax where you can expressly say "Run this test only if CONFIG_XIP=y", which would be cleaner.  I have to dig it up out of the sanitycheck source every time though because I can't remember how it looks.

Reply:
filters are annoying, for --all we would have to generate .config for all boards to know which boards we want to build for, take one hour in CI to just figure out which tests need to run (with --all).

Reply:
Ah, ok.  I was wondering if there was a better way to do this (I was thinking: inclusively, rather than exclusively, but I wasn't certain if the number of XIP platforms was limited).  I'll make a note to figure out the filter you mention and submit a patch to do it that way at some point in the future.